ament_flake8/ament_flake8/configuration/ament_flake8.ini

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = D100,D101,D102,D103,D104,D105,D203,D212,D404
+ignore = C816,D100,D101,D102,D103,D104,D105,D203,D212,D404
```

Comment:
is this ignored because it conflicts with another enforced rule (C815?) ?

Reply:
No, `C816` is a suggestion specific to Python 3.6+. Since we are targeting only 3.5 we can't use the suggested style.